Gemfile

```diff
@@ -29,7 +29,7 @@ gem 'mechanize'
 gem 'sass'
 gem 'jquery-rails'
 
-gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git', tag: 'v0.1.1'
+gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git'
```

Comment:
I read over [this discussion](https://github.com/bundler/bundler/issues/3697), and also this [bundler release notice](http://bundler.io/blog/2015/06/24/version-1-10-released.html#bundled-with). IMO this best thing to do is for all of us to upgrade to bundler 1.10.x. 1.5.3 is almost two years old; I don't know why it is OpsWorks' default. And on that point: I recently had to upgrade an OpsWorks stack from 1.5.3 to 1.10.6 in order to make the stack's deployment work with modern bundler syntax.